Digit_Count/Digit_Count.rb

```diff
@@ -0,0 +1,25 @@
+#Digit count of given number
+
+puts "Enter the number : "
+num=gets.chomp.to_i
+
+temp=num
+count=0
+
+#Loop to determine the number of digits
+while (temp>0)
+    count+=1
+    temp=temp/10
+end
+
+puts "The given number #{num} has #{count} digits in it"
+
+#SAMPLE INPUT AND OUTPUT
```

Comment:
please add a case for 0 as well. 
Your code will say 0 has 0 digits. 
Also, can you modify to add case for -ve numbers ?